docs/references/openapi-core.yaml

```diff
@@ -0,0 +1,346 @@
+openapi: "3.0.2"
+
+info:
+  title: Certifaction Core API
+  description: |
+    Certifaction Core-API is the central management system for our digital signing platform, enabling secure handling of users, organizations, roles and others.
```

Comment:
Certifaction Admin API is used to to manage users, roles and organisations.

Reply:
[3a83b9d](https://github.com/certifaction/cli/pull/44/commits/3a83b9d2f47de997a9afbbf6ad291d1e523aa156)